src/PrestaShopBundle/Resources/config/services/core/category.yml

```diff
@@ -18,3 +19,10 @@ services:
     public: false
     arguments:
       - !php/const _PS_CAT_IMG_DIR_
+
+  prestashop.core.category.cache.adapter:
```

Comment:
The idea is good, but if we start creating multiple cache services with each one having their own specific cache folders I'm araid clearing all of them efficiently may become a nightmare to maintain

I need to look into it, but I believe there is a system of Cache pool with SF, that allows you to have a common cache divied into separate sub pools It alllows you to fine tune clearing one sub cache at a time (when you modify the related domain) while still having one global cache that you can clear to clean everything

Reply:
Usually a good practive is to rely on the `CacheInterface` in the service that relies on it (not a specific filesystem), this way you can change the cache policy simply by changing your configuration and the software doesn't need to be updated

Wed don't have a clear cache policy in Symfony yet, but this idea deserves having a thought about it